Simple PSI example (#58)
intel/he-toolkit: README.md

  - [Examples](#examples)
    - [Secure Query](#secure-query)
    - [Logistic Regression](#logistic-regression)

>     - [Psi](#psi)

Reviewer: ```suggestion
    - [Private Set Intersection](#private-set-intersection)
```

[README](he-samples/examples/logistic-regression/README.md) for usage
information.

> ### Psi

Reviewer: ```suggestion
### Private Set Intersection
```

> The [psi](he-samples/examples/psi) example computes the intersection of two

Reviewer: ```suggestion
The [Private Set Intersection (PSI)](he-samples/examples/psi) example computes the intersection of two
```

> given sets. The program computes a hash value for each entry of both the client
> and the server sets, then using the HELib BGV scheme, it encrypts the client
> set and computes the intersation, so it returns all the encryted elements that
> are common to both sets. See the [README](he-samples/examples/psi/README.md)
> for usage information.

Reviewer: ```suggestion
given sets. The program computes a hash value for each entry of both the client
and the server sets, then using the HElib BGV scheme, it encrypts the client
set and computes the intersection, returning all the encrypted elements that
are common to both sets. See the [README](he-samples/examples/psi/README.md)
for usage information.
```
